torchdrivesim/simulator.py

```diff
@@ -213,6 +213,13 @@ def get_agent_size(self) -> TensorPerAgentType:
         """
         pass
 
+    @abc.abstractmethod
+    def get_agent_type(self) -> TensorPerAgentType:
+        """
+        Returns a functor of BxA long tensors representing agent type.
```

Comment:
This needs better documentation. What do those tensors index into?

Reply:
Updated. Let me know if it's better now.